definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -66,5 +66,13 @@
     "FieldType": "FixedList",
     "FieldTypeParameter": "ServiceApplicationPaymentTypeEnum",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServiceRefusalDraft",
+    "Label": "Service Refusal draft",
+    "FieldType": "Document",
+    "SecurityClassification": "Public"
```

Comment:
Adding definition to cater for generated draft refusal order. Examples are taken from the `DNRefusalDraft` definition